src/api/app/views/layouts/webui2/_personal_navigation.html.haml

```diff
@@ -0,0 +1,61 @@
+- if User.current && !User.current.is_nobody?
+  %li.nav-item
+    = link_to(home_path, id: 'link-to-user-home', class: 'nav-link') do
+      = User.current.login
+  %li.nav-item
+    - tasks = User.current.tasks
+    = link_to(user_tasks_path, class: 'nav-link') do
+      - if tasks > 0
+        %span{title: 'where an action is requested from you' }
+          = pluralize(tasks, 'Task')
+      - else
+        Tasks
+  %li.nav-item
+    - if User.current.home_project
+      = link_to(project_show_path(User.current.home_project), class: 'nav-link') do
+        Home Project
+    - else
+      = link_to(new_project_path(name: User.current.home_project_name), class: 'nav-link') do
+        Create Home
+  %li.nav-item
+    = link_to(session_destroy_path, method: :delete, id: 'logout-link', class: 'nav-link') do
+      Logout
+- elsif CONFIG['kerberos_mode']
+  = link_to 'Log In', session_new_path
+- else
+  - if CONFIG['proxy_auth_mode'] == :on
+    - unless CONFIG['proxy_auth_register_page'].blank?
```

Comment:
`if` + `present?` :pray: 

Why is HAML Lint not running [Rails/Blank cop](https://rubocop.readthedocs.io/en/latest/cops_rails/#railsblank)? :confused:

Reply:
@ana06

> Why is HAML Lint not running [Rails/Blank cop](https://rubocop.readthedocs.io/en/latest/cops_rails/#railsblank)? :confused: 

Rubocop is disabled in `.haml-lint_todo.yml` :sob: We should enable it, at least for the new files... :thinking:

Reply:
@Ana06, if is possible, we should enable it :smile_cat:

Reply:
@DavidKang 

> @Ana06, if is possible, we should enable it :smile_cat: 

it is... we just need to exclude all the files except the ones in the directory where the new files are. I would say this PR is the place to do so... :thinking:

Reply:
which introduces code with linting offenses which will be extra work when enabling the lint...

Reply:
Well the lint doesn't exist/work. So there aren't any offenses :man_shrugging:

Reply:
Do you guys really want to block all progress because of whitespace? Noone works on new webui features because it's unclear on what framework to put it. Lint the earth, but keep it rotating!

Reply:
writing `unless` + `blank?` is not a whitespace... it is more difficult to read and understand.. and it will be more difficult to change afterwards. This actually was the reason why it was disabled. When we tried to enabled, there were that many offenses that couldn't be autocorrected, that we just were not able to enable... The problem could be easily solved in the new views...

But this is just my opinion. If the rest of you don't think it is important, then merge it....

Reply:
yeah, problem can be solved afterwards by someone else - without this PR there won't be any new views to fix. You're replacing a parallalizable problem with a problem that has to be fixed serialized. Think big Ana!

Reply:
tbh, I don't understand why we didn't just fix this condition right away. After all it's just a 30 seconds fix:-)

The single offense get's fixed [here](#5146). For the rubocop enablement for haml files I created a [card](https://trello.com/c/QWsobuy1/1454-enable-rubocop-for-haml) in our backlog.